static/d3/d3_graph.js

```diff
@@ -0,0 +1,676 @@
+$( window ).load(function() {
+  $("#graph1").hide();
+  $("#graph1").css("background-color", "white");
+});
+
+function change_graph() {
```

Comment:
Does this function belong in d3/d3_graph.js? It looks like something that should belong in rage.js (because it refers to flot and d3).